src/components/shadows/examples/surface-shadows-bad-usage.tsx

```diff
@@ -0,0 +1,85 @@
+                        <p>
+                            Lorem Ipsum racconta una storia. Non come le altre.
+                            Un individuo che, in un giorno d'estate, s'imbatte
+                            in qualcosa di diverso dalla sua consuetudine. Egli
+                            intreccia la sua vita con quelle di altre persone
+                            del suo paese. Persone che hanno problemi di cuore,
+                            persone che, semplicemente, vorrebbero qualcuno con
+                            cui parlare.
+                        </p>
+                        <div class="box">
+                            <p>To look distinct, I only use a shadow effect.</p>
+                            <p>
+                                But that makes me look like a temporay popover
+                                or some sort of message. User may think I am
+                                covering some of the content.
+                            </p>
+                        </div>
+                        <p>
+                            Il tutto amalgamato in un viaggio introspettivo nei
+                            pensieri profondi dell'autore. Non è un romanzo.
+                            Questa storia è semplicemente la descrizione di ciò
+                            che lo scrittore si è trovato ad osservare con un
+                            taccuino tra le mani. Perché non c'è nulla di più
+                            affascinante della realtà.
+                        </p>
+                    </div>
+                </div>
+                <div class="do">
+                    <limel-header icon="ok" heading="Do"></limel-header>
+                    <div class="content">
+                        <p>
+                            Lorem Ipsum racconta una storia. Non come le altre.
+                            Un individuo che, in un giorno d'estate, s'imbatte
+                            in qualcosa di diverso dalla sua consuetudine. Egli
+                            intreccia la sua vita con quelle di altre persone
+                            del suo paese. Persone che hanno problemi di cuore,
+                            persone che, semplicemente, vorrebbero qualcuno con
+                            cui parlare.
+                        </p>
+                        <div class="box">
+                            <p>
+                                <limel-icon
+                                    name="information"
+                                    size="x-small"
+                                    badge={true}
+                                />
+                                To look distinct,
+                            </p>
+                            <p>
+                                I use a different background color, a better
+                                layout and enough space with my surrounding
+                                content.
```

Comment:
If you only fixed the typos and discarded my other suggestion above, this example is fine already.

If you did follow my suggestion to change from first person, then change this to 👇

```suggestion
                                To help distinguish an element,
                            </p>
                            <p>
                                use a different background color, a better
                                layout and enough space to the surrounding
                                content.
```